requirements.txt

```diff
@@ -10,6 +10,7 @@ markupsafe==1.1.1 # from jinja
 pyopenssl>=21.0.0,<23.0.0  # bullseye uses 20.0.1, but only the patched debian version still works.
 cryptography>=3.3.2 # for pyopenssl
 pyyaml==5.3.1
+pydantic==1.7.4
```

Comment:
don't forget to add pydantic to debian/control too

Reply:
Ah oui, good point. Thanks.